changelog/_unreleased/2025-07-10-add-error-handling-to-youtube-cms-element-link-parsing.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Add error handling to YouTube CMS element link parsing
+author: Lukas Völler
+author_email: voeller.lukas@gmail.com
+author_github: @https://github.com/LukasVoeller
+---
+# Core
+*
+___
+# API
+*
+___
+# Administration
+*
```

Comment:
Almost there: It's missing a short description of what you actually changed ([here's a recent example](https://github.com/shopware/shopware/blob/trunk/changelog/release-6-7-0-0/2025-06-05-fix-generating-variants-shows-nan.md))